cmd/opentelemetry-collector/app/defaults/default_config.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/exporter/cassandra"
 	"github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/exporter/elasticsearch"
+	"github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/exporter/grpc"
```

Comment:
minor, but think the exporter should be in package `grpc-plugin` to avoid confusion with the jaeger-grpc protocol exporter

Reply:
agree, I will refactor it